app/models/log_entry.rb

```diff
@@ -9,30 +9,6 @@ class LogEntry < ApplicationRecord
   belongs_to :member
 
   default_scope { order(created_at: :asc) }
-
```

Comment:
I added this as a helper method, although it could seem as this existed before (check [here](https://github.com/Samfundet/Samfundet/pull/590/files/)).